guides/spin/comparison/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title:  "Spin and Roer + Managed Pipeline Templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Foreward
```

Comment:
Consider an intro along the lines of:

"Users are often interested in learning about how to implement an 'as-code' approach for Spinnaker, whether in conjunction with automating onboarding for teams, or just managing and versioning Spinnaker configuration.

A related question that also comes up is what the recommended way is to interact with Spinnaker programmatically, especially with authentication is enabled.

This document intends to describe two Spinnaker features that usually feature in the responses to those two questions: Managed Pipeline Templates (MPT)/`roer`, and the new `spin` CLI.

..."